<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>551</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="clientGetEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="getButton_1">
       <property name="text">
        <string>CilentGet</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="serverGetEdit"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="getButton_2">
       <property name="text">
        <string>ServiceGet</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="clientSendEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clientSendButton">
       <property name="text">
        <string>ClientSend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="serverSendEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="serverSendButton">
       <property name="text">
        <string>ServiceSend</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
